static/index.html

```diff
@@ -9,7 +9,10 @@
   </head>
   <body>
     <div id="main">
-      <div class="loading"></div>
+      <div class="pop-throbber">
+        <div class="pop-throbber__message">Loading…</div>
+        <div class="pop-throbber__image--thinking"></div>
```

Comment:
This is annoying, but technically BEM requires that any modifier have a corresponding non-modified block/element. So for full compliance this should also have a class `pop-throbber__image`, even if that class isn’t referenced in the CSS. For simplicity, though, I’d suggest just getting rid of the modifier altogether, and making it implicit that the throbber’s image is a thinking-state Pop (which seems reasonable to assume).